recipes/pytfa/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
This isn't needed as it'll be found from the source tarball (which is preferable so it's always up to date).

You still need the `license_file` key in the `meta.yaml`.

Reply:
okey I see